Sources/MongoSwift/BSON/BsonEncoder.swift

```diff
@@ -522,10 +522,12 @@ private class MutableArray: BsonValue {
         try self.array.encode(to: storage, forKey: key)
     }
 
+    init() {}
```

Comment:
Is this required because `required init(from iter: DocumentIterator)` was added below and Swift would otherwise no longer create an empty initializer for us?